src/libs/ReportUtils.js

```diff
@@ -510,6 +510,16 @@ function navigateToDetailsPage(report) {
     Navigation.navigate(ROUTES.getReportParticipantsRoute(report.reportID));
 }
 
+/**
+ * Generate a random reportID between 98000000 (the number of reports before the switch from sequential to random)
+ * and the maximum safe integer of js (53 bits aka 9,007,199,254,740,991)
```

Comment:
NAB: It might be helpful to include a brief note here about why random is okay (e.g. "the likelihood of collision is blah blah")